corehq/apps/style/static/preview_app/less/preview_app/form.less

```diff
@@ -40,12 +40,7 @@
 }
 
 .form-single-question {
-  padding-top: 66px;
-  min-height: 443px;
-}
-
-.preview-tablet-mode .form-single-question {
-  min-height: 642px;
```

Comment:
it's so that the background color goes all the way to the bottom. I don't think this should be removed.